samples/src/main/java/org/opensearch/client/samples/Bulk.java

```diff
@@ -70,21 +73,46 @@ public static void main(String[] args) {
             BulkResponse bulkResponse = client.bulk(bulkReq.build());
             LOGGER.info("Bulk response items: {}", bulkResponse.items().size());
 
-            Query query = Query.of(qb -> qb.match(mb -> mb.field("title").query(fv -> fv.stringValue("Document"))));
-            final SearchRequest.Builder searchReq = new SearchRequest.Builder().allowPartialSearchResults(false)
-                .index(List.of(indexName))
-                .size(10)
-                .source(sc -> sc.fetch(false))
-                .ignoreThrottled(false)
-                .query(query);
-            SearchResponse<IndexData> searchResponse = client.search(searchReq.build(), IndexData.class);
+            // wait for the changes to reflect
+            Thread.sleep(3000);
```

Comment:
There is no need to wait, the bulk request does it already: `BulkRequest.Builder bulkReq = new BulkRequest.Builder().index(indexName).operations(ops).refresh(Refresh.WaitFor);

Reply:
That is a much cleaner solution, I'll get rid of the other wait and leverage the refresh there as well.